src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -303,7 +303,7 @@
         {% endif %}
     {%- endfor %}
 
-    {% if not form.methodRendered %}
+    {% if not form.methodRendered and form.parent is null %}
```

Comment:
It might break, if a field is named "parent". As discussed in https://github.com/symfony/symfony/pull/19492, I think we should be able to hardcode:
```twig
{% if not form.methodRendered and not form.hasParent %}
```
to know if a form view is a root.